plugins/outputs/d2l_kinesis/d2lKinesisOutput.go

```diff
@@ -275,7 +284,7 @@ func (k *d2lKinesisOutput) putRecords(
 
 	totalRecordCount := len(records)
 
-	entries := make([]*kinesis.PutRecordsRequestEntry, totalRecordCount)
```

Comment:
`PutRecordsRequestEntry` is really just 2 pointers and a `nil`, so they stopped doing the extra layer of pointer

https://github.com/aws/aws-sdk-go-v2/blob/c07a63e84ec500ace7fefbf5391b9028f7615bd7/service/kinesis/types/types.go#L152